### PR TITLE
QC: added details on cutadapt

### PR DESCRIPTION
**Pull request by bernt-matthias**
ping @bebatut: as discussed at EGD I added the algorithmic details of quality filtering of cutadapt. For the adapter alignment I added a short QA section.

### CODE REVIEW

File: topics/sequence-analysis/tutorials/quality-control/tutorial.md [hunk 1]
```diff
@@ -482,6 +482,42 @@ It is usual that the quality of the sequences is worse for the reverse than for
 >    {: .question}
 {: .hands_on}
 
+
+> ### {% icon comment %} Algorithmic details
+> 
+> One of the biggest advantage of cutadapt over other tools for trimming is that it is properly documented (https://cutadapt.readthedocs.io) which allows know how the tool works in detail.
```
Comment (bgruening) [inline topics/sequence-analysis/tutorials/quality-control/tutorial.md]: which allows you to know how ...?

File: topics/sequence-analysis/tutorials/quality-control/tutorial.md [hunk 2]
```diff
@@ -482,6 +482,42 @@ It is usual that the quality of the sequences is worse for the reverse than for
...
+
+> ### {% icon question %} Questions
+>
+> 1. What kind of alignment is used for finding adapters in reads. 
+> 2. What is the criterion to chose the best adapter alignment
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Semi global alignment, i.e., only the overlapping part of the read and the adapter sequence is used for scoring.
+> > 2. Analignment with maximum overlap is computed that has a smallest number of mismatches and indels.
```
Comment (bgruening) [inline topics/sequence-analysis/tutorials/quality-control/tutorial.md]: `An alignment`

File: topics/sequence-analysis/tutorials/quality-control/tutorial.md [hunk 3]
```diff
@@ -482,6 +482,42 @@ It is usual that the quality of the sequences is worse for the reverse than for
 >    {: .question}
 {: .hands_on}
 
+
+> ### {% icon comment %} Algorithmic details
```
Comment (bebatut) [inline topics/sequence-analysis/tutorials/quality-control/tutorial.md]: Can you use a detail box there? The detail box is expandable

Comment (bernt-matthias) [reply topics/sequence-analysis/tutorials/quality-control/tutorial.md]: sure

File: topics/sequence-analysis/tutorials/quality-control/tutorial.md (hunk 2 above)
Comment (bernt-matthias) [reply topics/sequence-analysis/tutorials/quality-control/tutorial.md]: done

File: topics/sequence-analysis/tutorials/quality-control/tutorial.md (hunk 1 above)
Comment (bernt-matthias) [reply topics/sequence-analysis/tutorials/quality-control/tutorial.md]: of course :)

**Pull request merged**